www/js/share/services/client.service.js

```diff
@@ -50,7 +50,7 @@ angular.module("binary").service("clientService", function(appStateService) {
     this.getExistingCurrencies = (accounts) => {
         const currencies = [];
         _.forIn(accounts, (account, key) => {
-            if (!/VR/i.test(account.id) && account.currency.length > 0) {
+            if (account.landing_company_name !== 'virtual' && account.currency.length > 0) {
```

Comment:
@nazaninreihani 
again, it's better to use `this.isAccountOfType` instead of checking manually.